addons/website_slides/tests/test_slide_channel.py

```diff
@@ -6,6 +6,41 @@
 from unittest.mock import patch
 
 
+class TestSlidesChannelMail(slides_common.SlidesCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.test_template_slides = cls.env['mail.template'].with_user(cls.user_manager).create({
+            'auto_delete': True,
+            'body_html': '<p>Fist fight with <t t-out="object.partner_id.name"/></p>',
```

Comment:
Is there a 'partner_id' field on slide ? And there is on test using it :disguised_face: